module/VuFind/src/VuFind/Controller/AbstractRecord.php

```diff
@@ -242,7 +242,7 @@ public function addtagAction()
         // Save tags, if any:
         if ($tags = $this->params()->fromPost('tag')) {
             $tagsService = $this->serviceLocator->get(TagsService::class);
-            $tagsService->addTagsToRecord(
+            $tagsService->linkTagsToRecord(
                 $driver,
                 $user,
                 $tagsService->parse($tags)
```

Comment:
Would it make sense to allow a string of tags as parameter to linkTagsToRecord? Then the parse method wouldn't need to be called here.